README.md

```diff
@@ -162,8 +162,8 @@ You can tune compression levels as you wish. Default compression levels are `1`.
 Configurable via `local.properties`:
 
 ```
-remote_build.local_gzip_level=1
```

Comment:
Well, it is, but at this point of time it should be very compatible for most users, so I would prefer to update only `minor` version